comfort-service.yml

```diff
@@ -0,0 +1,298 @@
+          - name: head_restraint
+            type: movement_t
+            description: |
+              The position of the head restraint 0 down, 1000 up
+
+      #
+      # Seat location
+      #
+      - name: seat_location_t
+        type: struct
+        description: |
+          The location of the seat in the vehicle
+        members:
+          - name: row
+            type: uint8
+            description: |
+              The row, front 0 and +1 toward rear
+          - name: index
+            type: uint8
+            description: |
+              The index within the row, 0 left most, +1 toward right
+
+      # Enums
+      #
+      #  TODO:
+      #  + Decide if we need to move enum_type (uint8) to a separate deployment
+      #    file, one per target (python, C++, etc).
+      #    The upside is a cleaner, more service-oriented catalog.
+      #    The downside is it inter-target operability is harder to
+      #    define an over-the-wire payload format since one target
+      #    can define uint16_t while another has uint8_t.
+      #
+      - name: seat_component_t
+        type: enumeration   # RESOLUTION: was enum. Ok to rename enumeration.
+
+        enum_type: uint8
+        options:
+          - name: base
+            value: 0
+          - name: cushion
+            value: 1
+          - name: lumbar
+            value: 2
+          - name: side_bolster
+
+            value: 3
+          - name: head_restraint
+            value: 4
+
+      - name: seat_t
+        type: struct
+        description: |
+          The structure used to describe the seat's position in the vehicle
+
+          members:
+          - name: location
+            type: seat_location_t
+            description: |
+              The location of the seat in the vehicle
+          - name: position
+            type: position_t
+            description: |
+              The various positions of the seat
+
+      # We can also redefine an existing type,
+      # effectively giving it a new name
+      # This is only an example and should be removed
+      - name: relative_movemenet_t
+        type: movement_t
+
+      # We can also rename a struct.
+      # This is only an example and should be removed
+      - name: relative_position_t
+        type: position_t
+
+
+    # METHODS
+    #
+    # This section defines all methods for the given namespace.
+    #
+    # A method invokes a command for a given service that may optionally
+    # return a value.
+    #
+    # Only one service instance will execute a call. The service discovery
+    # and invocation mechanism is up to the implementation to resolve (possibly
+    # with the help of deployment files).
+    #
+    methods:
+      - name: move
+        description: |
+          Set the desired seat position
+
+        in_arguments: # TODO FYI: Gunnar
+          - name: seat
+            description: |
+              The desired seat position
+            type: movement.seat_t
```

Comment:
What does "movement" refers to here? It is not namespace as that is "seats". Would "type: seat_t" suffice?